wordpress-mobile/WordPress-iOS
# Quick Start v2: Add new fancy alerts

Refs #10860

This updates the existing quick start alert to match mockups, and adds the new mockup to alert users that quick start has been updated:

Updated Alert | New Alert
---- | ----
![simulator screen shot - iphone xr - 2019-02-21 at 10 35 15](https://user-images.githubusercontent.com/517257/53207489-acd78100-35e8-11e9-8b03-3db39b94de3c.png) | ![simulator screen shot - iphone xr - 2019-02-21 at 10 55 11](https://user-images.githubusercontent.com/517257/53207498-b4972580-35e8-11e9-86a3-0a307ce06d71.png)


### To test:
- Create a site with v1 Quick Start†
- Launch this branch
- Ensure that you see the new upgrade alert
- Create a new site with this branch
- Ensure that you see the new alert asking for v2 of quick start

### †How to create a v1 site:
- You could turn off the `.quickStartV2` flag, then:
  - create a new site
  - **or** drop a `QuickStartTourGuide.find()?.setup(for: blog)` somewhere you can trigger easily


Update release notes:
